torch/nn/quantized/modules/functional_modules.py

```diff
@@ -0,0 +1,76 @@
+import torch
+from torch._ops import ops
+
+
+_VALID_OP_NAMES = {
+    'add',
+    'cat'
+}
+
+class FloatFunctional(torch.nn.Module):
```

Comment:
comments on this class? can be something simple explaining that it's a substitute for torch.*